### DISCUSSION TITLE
Malicious occupation

### DISCUSSION THREAD

**Discussion by ixyzorg** [npm/feedback#1113] [Registry]
There are many malicious people who occupy the name of the package, for whether this can delete its package, like the author of the ixyz package fssccf occupies a lot of packages

**Comment by leobalter** [comment]
If you think a name [infringes your trademark](https://docs.npmjs.com/policies/disputes#trademarks), please refer to our documentation on [name disputes](https://docs.npmjs.com/policies/disputes).
